doc/manual/noteworthy-differences.rst

```diff
@@ -6,6 +6,39 @@
 Noteworthy Differences from other Languages
 *******************************************
 
+Noteworthy and unusual Julia features
+---------------------------------
+
+* Julia allows identifiers with complex Unicode, for example you could represent the value of a chi-squared test with ``χ²``,
+  similarly ``χ⁽³⁾`` is used in electromagnetism for the third-order nonlinear susceptibility, another example is ``∇²``
+  which is the name of the Laplacian operator. All of these Unicode symbols are valid Julia identifiers.
+
+  It can also be useful for caching powers of ``x`` to ensure that intermediate products are reused optimally in time-critical code. 
```

Comment:
One would hope also that the compiler would be smart enough to lift things out of loops, if you have `x*x` or `x^2` inside a loop.  (If Jeff can't make the compiler do that, or more likely, has done it already, I'd eat my hat)